x-pack/platform/plugins/shared/cases/server/client/user_actions/users.ts

```diff
@@ -57,12 +57,14 @@ export const getUsers = async (
     const reporter = theCase.case.created_by;
     const reporterProfileIdAsArray = reporter.profile_uid != null ? [reporter.profile_uid] : [];
 
-    const userProfileUids = new Set([
-      ...assignedAndUnassignedUsers,
-      ...participantsUids,
-      ...assigneesUids,
-      ...reporterProfileIdAsArray,
-    ]);
+    const userProfileUids = new Set(
+      [
+        ...assignedAndUnassignedUsers,
+        ...participantsUids,
+        ...assigneesUids,
+        ...reporterProfileIdAsArray,
+      ].filter((uid) => uid !== '')
```

Comment:
nit: Should we also check for `null` or `undefined` like `filter(isEmpty) // lodash`?